tapioca.gemspec

```diff
@@ -33,10 +33,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency("parlour", ">= 2.1.0")
   spec.add_dependency("thor", ">= 0.19.2")
 
-  spec.add_development_dependency("bundler", "~> 1.17")
```

Comment:
Why do we need to move that out to the Gemfile?

Reply:
No special reason, but I don't see the point of dev-dependencies in a Bundler world, so I just moved all of them into the `Gemfile`. Basically, the `Gemfile` is only used during development (dev/test environments) so whatever it has are dev dependencies.

Reply:
It confused me because we still have a `development, test` group defined.

Reply:
True, we probably don't need those either. TBH, I don't think anyone is asking Bundler to load a `test` group either in dev nor in CI anyway, so that might be something we can remove actually.